sys/sys/event.h

```diff
@@ -163,6 +163,7 @@ struct kevent32_freebsd11 {
 #define EV_ENABLE	0x0004		/* enable event */
 #define EV_DISABLE	0x0008		/* disable event (not reported) */
 #define EV_FORCEONESHOT	0x0100		/* enable _ONESHOT and force trigger */
+#define EV_KEEPUDATA	0x0200		/* do not update the udata field */
```

Comment:
This probably deserves a manpage update mentioning the flag.

Reply:
That's absolutely fair.  I'll do that in the morning and push over this branch for merge.